src/audio/kpb.c

```diff
@@ -973,34 +1097,39 @@ static int kpb_copy(struct comp_dev *dev)
 			ret = PPL_STATUS_PATH_STOP;
 			break;
 		}
+		if (kpb->mic_sel_cnt_ == 0) {
```

Comment:
what does it mean if `mic_sel_cnt_ == 0`?

Reply:
The IPC large config set for microphone selection was not sent and kpb works as before - it simply copies the data.